examples/query-suggestions-history-with-category/app.tsx

```diff
@@ -0,0 +1,36 @@
+import { autocomplete } from '@algolia/autocomplete-js';
+import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
+import { createLocalStorageRecentSearchesPlugin } from '@algolia/autocomplete-plugin-recent-searches';
+import algoliasearch from 'algoliasearch';
+
+import '@algolia/autocomplete-theme-classic';
+
+const appId = 'latency';
+const apiKey = '6be0576ff61c053d5f9a3225e2a90f76';
+const searchClient = algoliasearch(appId, apiKey);
+
+const recentSearchesPlugin = createLocalStorageRecentSearchesPlugin({
+  key: 'qs-with-rs-example',
+  limit: 5,
+});
+
+const querySuggestionsPlugin = createQuerySuggestionsPlugin({
+  searchClient,
+  indexName: 'instant_search_demo_query_suggestions',
+  getSearchParams() {
+    return recentSearchesPlugin.data.getAlgoliaSearchParams({
+      hitsPerPage: 5,
+    });
+  },
+});
```

Comment:
This example aims at showcasing Recent Searches with categories. To demonstrate that, we need to add `categoryAttribute` to the QS plugin, so that the Recent Searches plugin displays them.

![image](https://user-images.githubusercontent.com/6137112/109809638-d8a28e00-7c28-11eb-9d87-2c618eb6bb4c.png)

You can then remove the Categories plugin.